README.rst

```diff
@@ -57,56 +41,41 @@ Here is a quick example of plotting an AIA image:
 
 .. code:: python
 
-    >>> import sunpy.map
-    >>> from sunpy.data.sample import AIA_171_IMAGE
-    >>> aia = sunpy.map.Map(AIA_171_IMAGE)
-    >>> aia.peek()
+   >>> import sunpy.map
+   >>> from sunpy.data.sample import AIA_171_IMAGE
+   >>>
+   >>> import matplotlib.pyplot as plt
+   >>>
+   >>> aia = sunpy.map.Map(AIA_171_IMAGE)
+   >>>
+   >>> aia.plot()
+   >>>
+   >>> plt.show()
 
 Getting Help
 ============
 
-For more information or to ask questions about SunPy, check out:
+For more information or to ask questions about ``sunpy`` or any other SunPy library, check out:
 
--  `SunPy Documentation`_
--  `SunPy Element Channel`_
--  `SunPy Mailing List`_
+-  `sunpy documentation <https://docs.sunpy.org/en/stable/>`__
+-  `SunPy IM channel <https://app.element.io/#/room/#sunpy:openastronomy.org>`__
```

Comment:
We also have this link twice we should use a reference for it so we only have it once.